src/v0/destinations/snapchat_conversion/data/OrderCompletedConfig.json

```diff
@@ -6,7 +6,7 @@
   },
   {
     "destKey": "item_category",
-    "sourceKeys": "properties.item_category",
+    "sourceKeys": ["properties.item_category", "properties.category"],
```

Comment:
going with this order to maintain already configured destinations as theey will be using this order